app/controllers/users.js

```diff
@@ -45,3 +46,14 @@ exports.signIn = async (req, res, next) => {
     return next(error);
   }
 };
+
+exports.getUsers = async (req, res, next) => {
+  try {
+    const pagination = mapperPagination(req.query.currentPage, req.query.perPage);
```

Comment:
you should schema validation for the pagination parameters, stating minimum and maximum values and other stuff